app/controllers/two-factor-auth-controller/post-configure-controller.js

```diff
@@ -21,16 +21,14 @@ module.exports = (req, res) => {
       return res.redirect(paths.user.profile)
     })
     .catch((err) => {
-      let errorMessageLog, errorMessageUser
-      if (err.errorCode === 401) {
-        errorMessageLog = `Activating new OTP key failed, bad code`
-        errorMessageUser = `<h2>There was a problem with the details you gave for:</h2><ul class="error-summary-list"><li><a href="#code">Please enter a valid security code</a></li></ul>`
+      let errorMessage
+      if (err.errorCode === 401 || err.errorCode === 400) {
+        errorMessage = `<h2>There was a problem with the details you gave for:</h2><ul class="error-summary-list"><li><a href="#code">Please enter a valid security code</a></li></ul>`
```

Comment:
It might be of help to someone having problems if the message told them what a valid security code looks like.

Reply:
Maybe, but maybe more for the page content rather than the error message.